src/esp/sensor/VisualSensor.cpp

```diff
@@ -22,5 +28,21 @@ void VisualSensor::bindRenderTarget(gfx::RenderTarget::uptr&& tgt) {
   tgt_ = std::move(tgt);
 }
 
+void VisualSensor::updateResolution(int height, int width) {
```

Comment:
For the setter and getter functions, we always use setSomething(...) or getSomething(...) as the function name.
Would you change it to `setResolution` 
More importantly, would you please pay attention to the reviewer's comments? I already mentioned it in the previous round of review.

Reply:
Sorry, I missed it in the previous round of review, but want to let you know that I appreciate each and every comment from reviewers! I wasn't able to address everything I wanted to in the last commit but wanted to get some big parts in to make sure I was on the right track.